templates/dashboard/customer/detail.html

```diff
@@ -133,8 +134,8 @@ <h5 class="data-table-title">
                 <div>{% trans "No default shipping address." context "Customer without default shipping address message" %}</div>
               {% endif %}
               <svg data-src="{% static "dashboard/images/email.svg" %}" width="14" height="14" />
-              <a href="mailto:{{ customer.email }}">
-                {{ customer.email }}
+              <a href="#">
+                {{ customer.email|obfuscate_string }}
```

Comment:
I think it would be more eye pleasant if we could append `@example.com` to the truncated version, so viewers will have a clue, what it should be in the first place.
eg. `test@gmail.com` => `t...@example.com`